core/src/lib.rs

```diff
@@ -51,3 +51,7 @@ pub use indexmap;
 pub use player::{Player, PlayerBuilder};
 pub use swf;
 pub use swf::Color;
+
+/// This file is built by 'core/src/avm2/playerglobal/compile_swc.sh'
+/// See that script, and 'core/src/avm2/playerglobal/README.md', for more details
+pub const RUFFLE_PLAYERGLOBAL_AVM2: &[u8] = include_bytes!("avm2/playerglobal/library.swf");
```

Comment:
Let's move this directly to `core/src/avm2/globals.rs`, since it's used only there. Also, "Ruffle" and "AVM2" don't contribute a lot to the meaning of this constant, as they're pretty obvious. I suggest leaving it just `PLAYERGLOBAL`. Same goes for `load_ruffle_playerglobal`, which can be renamed to simply `load_playerglobal`.

Reply:
The idea behind "ruffle" was to emphasize that this is our custom playerglobal, and not the original Flash playerglobal. However, I'm fine with renaming it